ansible/roles/postgresql/templates/pgbouncer.ini.j2

```diff
@@ -0,0 +1,207 @@
+;; database name = connect string
+;;
+;; connect string params:
+;;   dbname= host= port= user= password=
+;;   client_encoding= datestyle= timezone=
+;;   pool_size= connect_query=
+[databases]
+{% for db in postgresql_dbs | unique %}
+{{ db }} = host=localhost port=5432
```

Comment:
no, this is telling pgbouncer how to connect to the database. pgbouncer runs on port 6432 so we'll need to update our localsettings before we actually start using it